docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -18,6 +18,7 @@ default screen parts:
     % if get_config('debug'):
     `id: ${ user_info().question_id }`  
     `Package: ${ user_info().package } ${ package_version_number }; ${ al_version }`
+    <div data-variable="${ encode_name(user_info().variable) }" id="sought_variable" aria-hidden="true" style="display: none;"></div>
```

Comment:
I don't think `encode_name` handles `None` well, and `user_info().variable` is None if there's no variable being sought (purely informational screens? events?). Trying to figure out how to trigger it, but I think this will need to be caught and handled here.

https://github.com/jhpyle/docassemble/blob/10507a53d293c30ff05efcca6fa25f6d0ded0c93/docassemble_base/docassemble/base/functions.py#L4470

https://docassemble.org/docs/functions.html#user_info

Reply:
What about just `encode_name(str(user_info().variable)))`

Reply:
Thanks for catching this! The last of those suggestions is one I can implement, thanks!

Reply:
Hmm, even with an event screen, it looks like there's a variable being sought. Even so, better safe than sorry.

Reply:
I think the `None` would be triggered if it's a `mandatory` screen that isn't linked to a variable name. I have seen it before in our interviews though when testing the Github feedback form, so there must be another way to trigger it too.